tests/data/fixtures/pyrflx.py

```diff
@@ -187,6 +187,6 @@ def fixture_message_size_package(pyrflx_: pyrflx.PyRFLX) -> pyrflx.Package:
     return pyrflx_["Message_Size"]
 
 
-@pytest.fixture(name="message_size")
-def fixture_message_size(message_size_package: pyrflx.Package) -> pyrflx.MessageValue:
+@pytest.fixture(name="message_size_value")
+def fixture_message_size_valud(message_size_package: pyrflx.Package) -> pyrflx.MessageValue:
     return message_size_package["Msg"]
```

Comment:
Typo: fixture_message_size_valu**d**

Reply:
Fixed in https://github.com/Componolit/RecordFlux/pull/556/commits/02daff709bb35f85b83161d607a415c6ea2a7f13.